README.md

```diff
@@ -2,39 +2,101 @@
 
 <img align="right" src="doc/manual/figures/ale.gif" width=50>
 
-# Arcade-Learning-Environment
+# Arcade-Learning-Environment: An Evaluation Platform for General Agents
 
 The Arcade Learning Environment (ALE) -- a platform for AI research.
 
-## Quick start
 
-### To compile:
+This is the 0.5 release of the Arcade Learning Environment (ALE), a platform 
+designed for AI research. ALE is based on Stella, an Atari 2600 VCS emulator. 
+More information and ALE-related publications can be found at
+
+http://www.arcadelearningenvironment.org
+
+We encourage you to use the Arcade Learning Environment in your research. In
+return, we would appreciate if you cited ALE in publications that rely on
+it (BibTeX entry at the end of this document).
+
+Feedback and suggestions are welcome and may be addressed to any active member 
+of the ALE team.
+
+Enjoy,
+The ALE team
+
+===============================
+Quick start
+===============================
+
+Install main dependences:
+```
+sudo apt-get install libsdl1.2-dev libsdl-gfx1.2-dev libsdl-image1.2-dev cmake
+```
+
+Compilation:
 
 ```
 $ cmake -DUSE_SDL=ON -DUSE_RLGLUE=OFF -DBUILD_EXAMPLES=ON .
 $ make -j 4
 ```
 
-### To install python module:
+To install python module:
 
 ```
 $ pip install .
 or
 $ pip install --user .
 ```
 
-For more details and installation instructions, see the [website](http://www.arcadelearningenvironment.org) and [manual](doc/manual/manual.pdf). To ask questions and discuss, please join the [ALE-users group](https://groups.google.com/forum/#!forum/arcade-learning-environment).
+More instructions in the manual under doc/manual
+
+===============================
+List of command-line parameters
+===============================
+
+Execute ./ale -help for more details; alternatively, see documentation 
+available at http://www.arcadelearningenvironment.org.
+
+```
+-random_seed [n] -- sets the random seed; defaults to the current time
```

Comment:
These command line options are a bit outdated now (save perhaps for -config). @mcmachado Do you have a preference on keeping these here?